### ISSUE TITLE
Configure event trigger Google Calendar 

### ISSUE THREAD

**Issue by mrvirus9898** [ActoKids/AD440_W19_CloudPracticum#57]
Triggers will need to be set for the google calendar crawler. Please use Cloud Watch to create a trigger based on time, and on URL. You have already done something similar in #44 but this time work with Ryan to create several triggers in case he needs to break up his Google Crawler.

Please indicate the time spent on this, any issues that you are having, any good references you found for this subject, and credit anyone helped you out.

**Comment by TyReed12** [comment]
Update: Now have access to project AWS account but I do not currently have permissions for Cloudwatch in order to create the trigger. I need permission to call **CloudWatch Events API** so that I can create a trigger.

**Comment by TyReed12** [comment]
I seem to have all the permissions needed to complete my task. I'm currently waiting on Ryan's crawler code to be approved so that I can create the event trigger.

**Comment by TyReed12** [comment]
Successfully created trigger for GoogleCrawler on AWS.
![eventtriggerconfirm](https://user-images.githubusercontent.com/34496119/53132699-57866b80-3525-11e9-9065-f2817143ac68.JPG)

**Comment by TyReed12** [comment]
At this point the Trigger is created and will trigger the lambda function every 24 hours at 11:30 PM. @rberry206  is working on get his code into what is currently an empty lambda function that was just created a few minutes ago by the DevOps team.

Estimated time to be spent: 3 hours
Actual time spent: 3 hours.

**Comment by TyReed12** [comment]
![invocations](https://user-images.githubusercontent.com/34496119/53139147-90cad580-353d-11e9-94c5-11b7a1fa62ea.JPG)
